src/complex.rb

```diff
@@ -0,0 +1,74 @@
+class Complex
+
+  private
+
+  def get_cost_per_line(lang, cost_type = :all)
```

Comment:
Bliss.AI: report

| Debt | Issue |
|---:|---|
| 24.5 | Cyclomatic complexity for get_cost_per_line is too high. [51/6] |
| 24.5 | Perceived complexity for get_cost_per_line is too high. [52/7] |
| 7.4 | Assignment Branch Condition size for get_cost_per_line is too high. [89.31/15] |
| 4.5 | Method has too many lines. [55/10] |